resources/xlf/en/schema-compare.xlf

```diff
@@ -753,7 +753,7 @@
       <source xml:lang="en">default</source>
     </trans-unit>
   </body></file>
-  <file original="extensions/schema-compare/out/schemaCompareMainWindow" source-language="en" datatype="plaintext"><body>
+  <file original="extensions/schema-compare/out/schemaCompareResult" source-language="en" datatype="plaintext"><body>
```

Comment:
So I manually changed these xlfs to have the right name i.e. schemaCompareMainWindow but looks like I should have done this somewhere else since it is trying to recreate with old name. Is this to be corrected in mapping?